gio/src/auto/dbus_proxy.rs

```diff
@@ -935,9 +942,3 @@ impl<O: IsA<DBusProxy>> DBusProxyExt for O {
         }
     }
 }
-
-impl fmt::Display for DBusProxy {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
Ah that was already here. :laughing:

Reply:
Was just moved around for arbitrary reasons, yes